src/core_atmosphere/Registry.xml

```diff
@@ -1478,6 +1482,9 @@
                 <var name="xtime" type="text" dimensions="Time" units="YYYY-MM-DD_hh:mm:ss"
                      description="Model valid time"/>
 
+                <var name="Time" type="real" dimensions="Time" units="Seconds since YYYY-MM-DD hh:mm:ss"
```

Comment:
Since the attribute that's ultimately written to files uses `seconds since ...`, would it be more consistent to un-capitalize `Seconds` here?